connectors/src/lib/data_sources.ts

```diff
@@ -22,6 +22,8 @@ import logger from "@connectors/logger/logger";
 import { statsDClient } from "@connectors/logger/withlogging";
 import type { DataSourceConfig } from "@connectors/types/data_source_config";
 
+axios.defaults.timeout = 60000;
```

Comment:
I'm wondering if this does not change the timeout globally for all other axios requests being made from connectors 🤔. We should probably create an axios instance and change the timeout only for this one. Something like this:

```
const instance = axios.create();

instance.defaults.timeout = 2500;
```

Reply:
yeah you're right
will change in a followup pr